pkg/core/netmap/nodes.go

```diff
@@ -17,12 +17,15 @@ func (x Node) PublicKey() []byte {
 // IterateAddresses iterates over all announced network addresses
 // and passes them into f. Handler MUST NOT be nil.
 func (x Node) IterateAddresses(f func(string) bool) {
+	for _, addr := range (netmap.NodeInfo)(x).ExternalAddresses() {
```

Comment:
So firstly we are going to process external addresses everywhere, is it expected for intra-container communication? I thought "old" addresses are preferred for this purpose.